api/net/Areas/Services/Controllers/ContentController.cs

```diff
@@ -110,14 +125,25 @@ public IActionResult FindByUid([FromQuery] string uid, [FromQuery] string? sourc
     [ProducesResponseType(typeof(ContentModel), (int)HttpStatusCode.OK)]
     [ProducesResponseType(typeof(ErrorResponseModel), (int)HttpStatusCode.BadRequest)]
     [SwaggerOperation(Tags = new[] { "Content" })]
-    public IActionResult Add(ContentModel model)
+    public async Task<IActionResult> AddAsync(ContentModel model)
     {
-        var result = _contentService.AddAndSave((Content)model);
-        return new JsonResult(new ContentModel(result));
+        var content = _contentService.AddAndSave((Content)model);
+
+        if (!String.IsNullOrWhiteSpace(_kafkaOptions.IndexingTopic))
```

Comment:
Need to index the content immediately.  This is why we're missing so much content.